content/4_Gold/Divis.mdx

```diff
@@ -488,34 +485,6 @@ $$
 \gcd(a_1, a_2, a_3, a_4) = \gcd(a_1, \gcd(a_2, \gcd(a_3, a_4))).
 $$
 
-<LanguageSection>
```

Comment:
why is this section deleted??

Reply:
Because the "solution" is wrong iirc. @SansPapyrus683 could you comment this out instead of deleting it for now please?

Reply:
ok sure